handbook/product/design/metrics/amplitude-for-designers.md

```diff
@@ -0,0 +1,89 @@
+# Amplitude for designers
+
+Amplitude provides session-based event data of Sourcegraph cloud features. This guide describes several key concepts and provides a tutorial on how designers can use Amplitude in the discovery process in addition to measuring results.
```

Comment:
Can we explain what session-based means here or under key concepts? I'm personally not clear on what that phrase means. Does session just mean a time-boxed instance of a user using a product? Is every event contained within a session?